### ISSUE TITLE
Why do I reproduce your code, but my accuracy rate is not as high as yours

### ISSUE THREAD

**Issue by feiyangdezhu** [1991viet/Character-level-cnn-pytorch#1]

**Comment by 1991viet** [comment]
Hi, 
Could you provide more information, e.x which dataset you use? Do you change any parameters? How big the difference between my accuracy and yours is ?
Bests,

**Comment by feiyangdezhu** [comment]
My training data was yelp_review_polarity_csv with the highest accuracy rate of 87%

**Comment by feiyangdezhu** [comment]
I didn't change any parameters

**Comment by 1991viet** [comment]
Also Large network (1024 channels), Adam optimizer ?

**Comment by feiyangdezhu** [comment]
Small network (256 channels),Adam optimizer

**Comment by feiyangdezhu** [comment]
Indeed,I'm very interested in NLP.And  I will also take a look at your very deep CNN and Hierarchical attention network repos. I'm looking forward to see how the loss decreases.Look forward to your work after two weeks.
